### DISCUSSION TITLE
Free Whisper-based web app with Transcript Editor

### DISCUSSION THREAD

**Discussion by montagao** [openai/whisper#1565] [Show and tell]
Feel free to check it out at https://translate.mom!

It uses the large-v2 model and includes a subtitle editor so you can edit any inaccuracies and inconsistencies before exporting the subtitles.

## Screenshot:
<img width="1660" alt="Screenshot 2023-08-01 at 9 01 59 AM" src="https://github.com/openai/whisper/assets/9814181/da440dfa-f5d7-41c5-9591-5ca7dc2492dd">

## Soon to be added:
- deepL support
- VAD filter support
- font customization

**Comment by LostArrows27** [comment]
does your website share source code, or at least can i ask you some question about this because i am struggling with the transcript timestamp using javascript and not know much about Python
